scripts/results_generation/compare.py

```diff
@@ -0,0 +1,102 @@
+import json
+import logging
+import statistics
+from pathlib import Path
+
+import numpy as np
+
+from team.dmp_trajectory import DmpTrajectory
+from team.dynamical_movement_primitives import DynamicMovementPrimitives
+from team.trajectory_base import TrajectoryBase
+
+
+def comparison_setup(regression: TrajectoryBase, reproduction: TrajectoryBase) -> None:
+    """
+    Manipulate regression and reproduction trajectories to bring them at same length
+    1. Trim initial static portion of both trajectories, not moving data
+    2. Check which trajectory is longer
+    3. Pad shorter trajectory to same length
+
+    :param regression: regression trajectory
+    :param reproduction: reproduction trajectory
+    :return: list with processed trajectories ready to be compared
+    """
+
+    if regression.is_longer(reproduction):
+        final_len = len(regression)
+        reproduction.pad_end_to(final_len)
+        return
+
+    final_len = len(reproduction)
+    regression.pad_end_to(final_len)
+
+
+def get_score_folder(
+    data_path: Path,
+    filename: str,
+    rewrite_gmcc: bool = False,
+) -> tuple[float, float, float, float]:
+    """Get the gmcc mean and joint errors (with std for both) of a task
+
+    :param data_path: a path to the data
+    :param filename: filename of reproductions
+    :param rewrite_gmcc: if True the gmcc is recalculated, if False, if it was
+    calculated before it is not recalculated. default to False because gmcc calculation
+    is slow
+    :return: mean and standard deviation of gmcc, mean and std of joint errror
+    """
+
+    logging.basicConfig(force=True)
+    team_logger = logging.getLogger("team")
+    team_logger.setLevel(logging.DEBUG)
+
+    dmp = DynamicMovementPrimitives.load_dmp(data_path)
```

Comment:
I believe we should use load method since is the one that is backward compatible with the type of file we are trying to load :)